### PR TITLE
prepare changelog for v0.17.1

### PR DESCRIPTION
**Pull request by vinay-gopalan**

### CODE REVIEW

**Comment by Zlaticanin** [comment]
There is another [PR](https://github.com/hashicorp/vault-plugin-secrets-azure/pull/186) open by @tomhjp. Should we combine the two?

**Comment by vinay-gopalan** [comment]
Ah thanks for pointing that out, completely missed that! Going to close this and add a comment on #186 to include the updates from here

**Comment by vinay-gopalan** [comment]
We have decided to move forward with the v0.17.1 release with just the panic fix in #191. This PR has been reopened for review

**Review by Zlaticanin — APPROVED** [review]

**Review by tomhjp — APPROVED** [review]
LGTM, just one optional suggestion on phrasing

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## Unreleased
+
+## v0.17.1
+
+BUG FIXES:
+* Add nil check for response when unassigning roles [[GH-191]](https://github.com/hashicorp/vault-plugin-secrets-azure/pull/191)
```
Comment (tomhjp) [inline CHANGELOG.md:6]: Looks like my comment got lost somehow. The suggestion I had meant to leave was to phrase this more directly in terms of impact users will see, e.g. "Avoid panic when unassigning roles", but it's very minor and not worth opening up another PR for 👍 I only wanted my review comment to make sense :-)

**Pull request merged**